Formula/snobol4.rb

```diff
@@ -1,9 +1,8 @@
 class Snobol4 < Formula
   desc "String oriented and symbolic programming language"
-  homepage "https://www.snobol4.org/"
-  url "ftp://ftp.ultimate.com/snobol/old/snobol4-1.5.tar.gz"
-  mirror "https://src.fedoraproject.org/lookaside/pkgs/snobol/snobol4-1.5.tar.gz/54ac3ddd51fb34ec63b1eb0ae7b99794/snobol4-1.5.tar.gz"
-  sha256 "9f7ec649f2d700a30091af3bbd68db90b916d728200f915b1ba522bcfd0d7abd"
+  homepage "https://www.regressive.org/snobol4/"
+  url "https://ftp.regressive.org/snobol/snobol4-2.3.1.tar.gz"
+  sha256 "91244d67d4e29d2aadce5655bd4382ffab44c624a7ea4ad6411427f3abf17535"
```

Comment:
Is there any documentation somewhere of the hosting change?

Reply:
There's a one line mention in the distribution History (change log) file:
3/25/2022:
        LOTS: switch to regressive.org for web & mail

Reply:
But that's on this new website right? So that's not really reliable for this.

Reply:
Correct.  It wasn't a planned transition.  One of the links on wikipedia has been fixed, but there are a few broken ones left :-/

Reply:
Is there any external source that already verified the change?